src/config.ts

```diff
@@ -51,6 +51,18 @@ export const WECHATY_PUPPET_SERVICE_ENDPOINT = () => {
   return undefined
 }
 
+export const GET_CHATIE_ENDPOINT_LIST = () => {
+  const privateEndpointList = process.env.PRIVATE_ENDPOINT_LIST // e.g. https://aaa.aaa,http://bbb.bbb
```

Comment:
Actually this is a list of service discovery endpoints. `PRIVATE_ENDPOINT_LIST` seems a little bit vague. So how about changing this env var to `SERVICE_DISCOVERY_ENDPOINT_LIST`? Or `SERVICE_DISCOVERY_ENDPOINTS`?